src/Viewer/components/Monaco/utils.ts

```diff
@@ -60,6 +59,19 @@ const initMonacoEditor = (
         }
     );
 
+    let resizeTimeout: NodeJS.Timeout | null = null;
+    const resizeObserver = new ResizeObserver(() => {
+        if (null !== resizeTimeout) {
+            console.log("canceled");
+            clearTimeout(resizeTimeout);
+        }
+        resizeTimeout = setTimeout(() => {
+            editor.layout();
+        }, 250);
```

Comment:
Shall we make it a const to avoid magic numbers? e.g., `MONACO_RESIZE_DEBOUNCE_TIMEOUT`.